.secrets.baseline

```diff
@@ -128,10 +128,12 @@
     {
       "path": "detect_secrets.filters.regex.should_exclude_file",
       "pattern": [
-        ".secrets.baseline"
+        ".secrets.baseline",
+        "src/syntheticdata-mcp-server/tests/test_constants.py",
+        "src/syntheticdata-mcp-server/tests/conftest.py"
       ]
     }
   ],
   "results": {},
```

Comment:
After running `detect-secrets audit .secrets.baseline`:
```suggestion
        ".secrets.baseline"
      ]
    }
  ],
  "results": {
    "src/syntheticdata-mcp-server/tests/test_constants.py": [
      {
        "type": "Secret Keyword",
        "filename": "src/syntheticdata-mcp-server/tests/test_constants.py",
        "hashed_secret": "da5bcedede3c8c32004d394af1c3e14eb05b6a45",
        "is_verified": false,
        "line_number": 13,
        "is_secret": false
      }
    ]
  },
```
You can update the `generate_at` date too...